src/common/log/logHelper.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/**
+ * Helper for the log utility.
+ */
+
+import * as util from "util";
+import {InternalError, InternalErrorLevel} from "../error/internalError";
+
+export enum LogLevel {
+    None = 0,
+    Error = 1,
+    Warning = 2,
+    Info = 3,
+    Debug = 4,
+    Trace = 5
+}
+
+export class LogHelper {
+    public static MESSAGE_TAG: string = "[vscode-react-native]";
+    public static INTERNAL_TAG: string = "[Internal]";
+    public static ERROR_TAG_FORMATSTRING: string = "[Error : %s] ";
+    public static WARN_TAG: string = "[Warning]";
+    private static ERROR_CODE_WIDTH: string = "0000";
+    private static LOG_LEVEL_NAME: string = "RN_LOG_LEVEL";
+
+    public static get logLevel(): LogLevel {
+        let valName: string = process.env[LogHelper.LOG_LEVEL_NAME];
+
+        if (typeof(valName) === "undefined") {
+            valName = "Info"; // Set the default LogLevel to LogLevel.Info
+        }
+
+        return (<any> LogLevel)[valName];
+    }
+
+    public static set logLevel(level: LogLevel) {
+        if (!level) {
+            return;
+        }
+
+        // Set the process env value
+        process.env[LogHelper.LOG_LEVEL_NAME] = LogLevel[level];
+    }
+
+    /**
+     * Gets the message of a non null error, if any. Otherwise it returns the empty string.
+     */
+     public static getErrorString(e: any): string {
+
+        if (e.isInternalError) {
+            let errorMessage = e.message;
+            let errorMessagePrefix = LogHelper.getErrorMessagePrefix(e);
+            return errorMessagePrefix + " " + errorMessage;
+       } else {
+            try {
+                return JSON.stringify(e);
```

Comment:
Right now this will try to JSON.stringify an Error object, but most of the interesting properties are non-enumerable so this won't work right.